lib/tasks/links.rake

```diff
@@ -0,0 +1,58 @@
+namespace :links do
+  desc "Remove all links of given type to given target document"
+  task :remove_from_all_documents, %w[target_content_id link_type] => :environment do |_, args|
+    raise "Missing parameter: target_content_id" unless args.target_content_id
+
+    target_content_id = args.target_content_id
+
+    raise "Missing parameter: link_type" unless args.link_type
+
+    link_type = args.link_type
+
+    link_sets = LinkSet.joins(:links).where(links: { target_content_id: target_content_id, link_type: link_type })
```

Comment:
Actually looking at @vanitabarrett's rake task: https://github.com/alphagov/content-tagger/compare/find-business-finder-related-links, we could use ```publishing_api.get_linked_items``` which means we don't have to do raw database access and we can go through the external api which may be a bit better